docs/markdown/redbox.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Redbox
+description: You can use a redbox when you need a beautiful box that's red. Draws attention from users, and bulls. Look closer and you might find a DVD
+badge: pilot
+component: true
```

Comment:
Just marking a component page as a component. Need to do this because of width constraints. A component page takes up full-width

Reply:
@rlingineni Are we still considering markdown for component pages or just for readme-like "getting started" pages?

Reply:
@AlbertCarreras  - wanted to keep this one around since it opens the door to supporting it in the future if we're open to it, and an example of how we can do it. A little easter egg. I can probably throw it out of the static paths, if we don't want to publish it. Maybe we can chat about in the next eng weekly, and if there is a world where this saves 10x the time of writing a regular page, then maybe it's worth keeping